tests/rest_api/test_0004_tasks.py

```diff
@@ -0,0 +1,183 @@
+
+
+class TestPostTasks:
+    _ADMIN = 'admin1'
+    _SAND_PROJ_ID = 1
+    _SAND_PROJ_OWNER = 'business1'
+    _SAND_PROJ_NOT_OWNER = 'business2'
+    _ORG = 'org2'
+    _ORG_PROJ_ID = 2
+    _ORG_SUPERVISOR_NOT_OWNER = 'user3'
+    _ORG_WORKER_NO_OWNERSHIP = 'worker3'
+
+    def _assign_user_to_proj(self, user, project_id, **kwargs):
+        response = patch_method(self._ADMIN, f'projects/{project_id}', { 'assignee_id': user['id'] }, **kwargs)
+        assert response.status_code == HTTPStatus.OK
+
+    def _test_can_create_task_in_project(self, user, spec, **kwargs):
+        response = post_method(user, '/tasks', spec, **kwargs)
+        print(response.content)
+        assert response.status_code == HTTPStatus.CREATED
+
+    def _test_cannot_create_task_in_project(self, user, spec, **kwargs):
+        response = post_method(user, '/tasks', spec, **kwargs)
+        assert response.status_code == HTTPStatus.FORBIDDEN
+
+    # [sandbox] admin can create task in project even he has no ownerships in this project
+    def test_admin_can_create_task_in_project(self):
+        spec = {
+            'name': 'test admin to create a task within a project',
+            'project_id': self._SAND_PROJ_ID,
+        }
+        self._test_can_create_task_in_project(self._ADMIN, spec)
+
+    # [sandbox] business cannot create task in project if he has no ownerships in this project
+    def test_business_cannot_create_task_in_project_without_ownership(self):
+        spec = {
+            'name': 'test business cannot create task without ownership',
+            'project_id': self._SAND_PROJ_ID,
+        }
+        self._test_cannot_create_task_in_project(self._SAND_PROJ_NOT_OWNER, spec)
+
+    # [sandbox] user that has one of these ownerships: [Project:owner, Project:assignee] and has less than 10 task can create task in project
+    def test_entitled_user_can_create_tasks_in_project(self, users):
+        spec = {
+            'name': 'test entitled user can create tasks in project',
+            'project_id': self._SAND_PROJ_ID,
+        }
+        self._test_can_create_task_in_project(self._SAND_PROJ_OWNER, spec)
+
+        user1 = list(filter(lambda x: x['username'] == 'user1', users))[0]
+
+        self._test_cannot_create_task_in_project(user1['username'], spec)
+        self._assign_user_to_proj(user1, self._SAND_PROJ_ID)
+        self._test_can_create_task_in_project(user1['username'], spec)
+
+    # [organization] supervisor can create task in project even he has no ownerships in this project
+    # TODO: SUPERVISOR DOES NOT SEE CREATED PROJECTS AND CANNOT CREATE TASKS
+
+    # def test_supervisor_can_create_task_in_project_without_ownership(self, users):
+    #     spec = {
+    #         'name': 'test supervisor can create task in project',
+    #         'project_id': self._ORG_PROJ_ID,
+    #     }
+    #     self._test_can_create_task_in_project(self._ORG_SUPERVISOR_NOT_OWNER, spec, org=self._ORG)
```

Comment:
@kirill-sizov Please, check the specification

Reply:
`worker3` has privilege `worker`, but `worker` (as privilege) cannot create a task in this case. To resolve this problem need to change DB (for example change `user3` role from `supervisor` to `worker`) and after that use `user3` in these tests.

Reply:
Yes, I was wrong this comment belongs to [this](https://github.com/openvinotoolkit/cvat/blob/2cc26a30e42ed1ba3bf4b4870f18e1cc80ee274a/tests/rest_api/test_0004_tasks.py#L169-L181) test